server/src/main/java/io/spine/server/ServerEnvironment.java

```diff
@@ -72,30 +76,24 @@
      * initialized with the {@linkplain Delivery#local() local} delivery by default.
      */
     @SuppressWarnings("FieldAccessedSynchronizedAndUnsynchronized")
-      // No synchronization in `reset()` as it is test-only.
+    // No synchronization in `reset()` as it is test-only.
     private @MonotonicNonNull Delivery delivery;
```

Comment:
Let's try to make the `Delivery` specific to the environment as well.

To do so, `configureDelivery` should be deprecated and migrated to `use(...)`.

Reply:
Just something I've found: for any non-`Tests` environment, if we want to configure a delivery for it, we also need to configure a storage factory for it first. Should we specify it in Javadocs?